Libraries/xcsdk/Sources/SDK/Manager.cpp

```diff
@@ -123,6 +123,27 @@ executablePaths() const
     };
 }
 
+std::vector<std::string> Manager::
+executablePaths(std::vector<xcsdk::SDK::Toolchain::shared_ptr> &toolchains, xcsdk::SDK::Target::shared_ptr &target) const
+{
+    std::vector<std::string> paths;
+
+    for (Toolchain::shared_ptr const &toolchain : toolchains) {
+        std::vector<std::string> toolchainPaths = toolchain->executablePaths();
+        paths.insert(paths.end(), toolchainPaths.begin(), toolchainPaths.end());
+    }
+
+    if (target != nullptr) {
+        std::vector<std::string> targetPaths = target->executablePaths();
```

Comment:
Ah, thanks for catching.  I figured the given structure was really weird before -- when grabbing the target's executable paths, it seemed weird that you could override the toolchains that the SDK target used (meaning that the paths returned weren't really associated with the SDK target).  But, if that's intended, I'll rethink.